src/urbit/app/chess.hoon

```diff
@@ -658,12 +810,47 @@
               ?~  new.move-result
                 [cards.move-result this]  ::  nice try, cheater
               =,  u.new.move-result
-              :-  cards.move-result
-              ?.  ?=(~ result.game)
+              ::
+              ::  XX: shouldn't notify if %chess is open
```

Comment:
I don't actually know if there's a way to avoid this, since the agent can't know if the frontend is open or not. The first thing that comes to mind is that the notifications still get sent, but opening the window/handling the update in the frontend using React must mark the notifications as seen? I imagine that this is what `%pals` does. Based on the observed behaviour of `%landscape`, I think that it may mark the notification path as deleted, so they don't even appear in Grid.

`~datder-sonnet`, `~palfun-foslup`, `~nocsyx-lassul`, or `~hastuc-dibtux` might be good people to ask about this.